server/parsec/events.py

```diff
@@ -338,9 +338,17 @@ class EventOrganizationExpired(BaseModel):
     organization_id: OrganizationIDField
 
 
-class EventUserRevoked(BaseModel):
+class EventUserRevokedOrFrozen(BaseModel):
```

Comment:
Why merge both events?
Defining two separate events seems more convenient in the long run

Reply:
This is to stay KISS: this event is only needed to indicate a user should no longer be allowed.

It is unlikely we need to differentiate further, and even if that's the case it's better to add a field to this event instead of splitting it into two the events (as this would open the door of bugs where user freeze is overlooked and only user revoked event is handled)

Reply:
I hear your point about the bug but it still feels contrived to me. IMO different events should be different classes, which seems pretty simple if KISS is your concern. I won't fight over it though

Reply:
Just for my general culture: are these events internal to the server? or they are meant to be consumed by some other component?

Reply:
> are these events internal to the server? 

Those events are internal to parsec server, so we can change them whenever we want with no backward compatibility whatsoever ;-)


> IMO different events should be different classes

I agree in theory, but here I see it as a very high risk of forgetting the user freeze given how niche this feature is

BTW this is not that different than the events we send when a new certificate is created: the event structure is defined according to what it is going to be consumed for, not what it comes from (hence we don't have one event per action creating a certificate, but an event if a common certif has been added, another one for realm certifs and another one for shamir)

This approach works fine here given our events are basically always single purposed (it wouldn't be the right approach if we had a lot of frequently changing logic reacting to various combination of events)